src/language/dafnyInstallation.ts

```diff
@@ -89,15 +89,41 @@ export async function getCompilerRuntimePath(context: ExtensionContext): Promise
   return configuredPath;
 }
 
+// We cache the language server runtime path so that we don't need to copy it every time.
+let LanguageServerRuntimePath: string | null = null;
+
 export async function getLanguageServerRuntimePath(context: ExtensionContext): Promise<string> {
-  const configuredPath = await ifNullOrEmpty(
-    getConfiguredLanguageServerRuntimePath(),
+  if(LanguageServerRuntimePath != null) {
+    return LanguageServerRuntimePath;
+  }
+  const configuredLanguageServerPath = getConfiguredLanguageServerRuntimePath();
+  let mightBeRecomputed = false;
+  if(configuredLanguageServerPath !== '' && configuredLanguageServerPath != null) {
+    mightBeRecomputed = true;
+  }
```

Comment:
You may want to extract the empty string check into a separate function. IIRC, we're doing this in multiple occasions so there might be an opportunity to introduce/re-use a common function.

Moreover, you could simplify the expression like this:
```suggestion
  let mightBeRecomputed = configuredLanguageServerPath !== '' && configuredLanguageServerPath != null;
```